app/jobs/deposit_job.rb

```diff
@@ -3,9 +3,10 @@
 # Performs a deposit (without SDR API).
 class DepositJob < ApplicationJob
   # @param [WorkForm] work_form
+  # @param [Work] work
   # @param [Boolean] deposit if true, deposit the work; otherwise, leave as draft
-  def perform(work_form:, deposit:)
-    cocina_object = ToCocina::Mapper.call(work_form:, source_id: source_id)
+  def perform(work_form:, work:, deposit:)
+    cocina_object = ToCocina::Mapper.call(work_form:, source_id: "h3:object-#{work.id}")
     # Until we're also doing updates:
```

Comment:
Not necessarily for this PR, but I wonder if we want to add some kind of specific text (i.e. "TODO") to comments like this, that are very helpful, to make sure we don't forget about areas of code that we anticipate removing or know are temporary.